torch/onnx/utils.py

```diff
@@ -1314,6 +1314,70 @@ def _validate_dynamic_axes(dynamic_axes, model, input_names, output_names):
             dynamic_axes[key] = value_dict
 
 
+# This function export torch::jit::Graph object
+# to serialized ONNX ModelProto.
+# In addition to the Python tests of this function
+# (found by searching
+# _export_jit_graph_to_onnx_model_proto in Pytorch
+# codebase), this function can be called in C++ to
+# to generate ModelProto without accessing disk.
```

Comment:
Rather than "without accessing disk", can you please explain how this function differs from the most similar other functions (maybe `_model_to_graph`?) and why those differences are useful.

Reply:
editted to be
```
    This function exports torch::jit::Graph object
    to serialized ONNX ModelProto.
    This function is for testing purpose.
    It only keeps the essential parts for IR graph conversions.
    It also does not interact with actual PyTorch modules nor
    PyTorch tensor inputs.
```

And I moved it to the test file instead of utils.